### ISSUE TITLE
Non chromium edge

### ISSUE THREAD

**Issue by ToddRB** [rosolko/WebDriverManager.Net#51]
Hi, the latest version gets the edge driver URLS from the new pool of urls...
https://msedgedriver.azureedge.net/ 

But I still need to access the original ones. I'm just using a direct download link in SetupDriver, that I determine by the OS version. 
new DriverManager().SetUpDriver(
                GetMicrosoftEdgeDownloadLink(version),
                Path.Combine(Directory.GetCurrentDirectory(), "Edge", GetEdgeWebDriverVersion(version), "X64", "MicrosoftWebDriver.exe"),
                "MicrosoftWebDriver.exe"
            );

Ie, I end up passing in 
"https://download.microsoft.com/download/3/4/2/342316D7-EBE0-4F10-ABA2-AE8E0CDF36DD/MicrosoftWebDriver.exe"

So basically, in the BinaryService.cs, I need it to copy the file over when it is an exe. Currently it only copies it to my working directory if its a .zip, and goes through the Unzip method. 

Thanks!

**Comment by rosolko** [comment]
@ToddRB Ok, I'll add support of legacy edge soon

**Comment by rosolko** [comment]
@ToddRB Available in 2.7.0. Use `LegacyEdgeConfig` for non chromium edge

**Comment by ToddRB** [comment]
That was fast! Thanks!
